Sprint-2/implement/vat.js

```diff
@@ -8,3 +8,11 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+
+function addVat(price){
+    return `${price * 1.2}`;
```

Comment:
Are the literals needed here?

Reply:
You are right. I didn't need to use literals here. That's a great point. Because we use a template literal when we need to include variables or expressions inside a string. Thank you.